app/assets/stylesheets/blacklight_gallery/_masonry.scss

```diff
@@ -0,0 +1,44 @@
+.view-type-masonry {
```

Comment:
Can you be more specific?  TBH, I never know when to use the Bootstrap variables vs. when we want something to stand-alone if something in Bootstrap changes.  I'm pretty much taking these colors from somebody else and I don't know if they intend to pin to the variables or not.